.github/actions/setup/action.yml

```diff
@@ -37,6 +37,11 @@ runs:
         uv pip install "numpy<2"
       shell: bash
 
+    - name: Install pandas
+      run: |
+        uv pip install "pandas<2.2.3"
```

Comment:
where is this used and what functionality can we get without it?

Reply:
i think this is resolved